examples/webgl_geometry_colors_lookuptable.html

```diff
@@ -34,287 +34,266 @@
 	<body>
 
 		<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - lookuptable - vertex color values from a range of data values.<br />
-		press A: change color map, press S: change numberOfColors, press D: toggle Legend on/off, press F: change Legend layout<br />
 		</div>
 
 		<div id="container"></div>
 
 		<script src="../build/three.js"></script>
 		<script src="js/math/Lut.js"></script>
 		<script src="js/WebGL.js"></script>
-		<script src="js/libs/stats.min.js"></script>
+		<script src="js/controls/OrbitControls.js"></script>
+		<script src="js/libs/dat.gui.min.js"></script>
 
 		<script>
-
 			if ( WEBGL.isWebGLAvailable() === false ) {
 
 				document.body.appendChild( WEBGL.getWebGLErrorMessage() );
 
 			}
 
-			var container, stats;
-
-			var camera, scene, renderer, lut, legendLayout;
+			var container;
 
-			var mesh, material;
+			var perpCamera, orthoCamera, renderer, lut;
 
-			var colorMap;
-			var numberOfColors;
+			var mesh;
+			var scene, uiScene;
 
-			var loader = new THREE.BufferGeometryLoader();
+			var params, verticalLegendGroup, horizontalLegendGroup;
+			var updateColorEvent;
 
 			init();
-			animate();
 
 			function init() {
 
 				container = document.getElementById( 'container' );
 
-				// SCENE
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0xffffff );
 
-				// CAMERA
-				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 100 );
-				camera.position.set( 0, 0, 10 );
+				uiScene = new THREE.Scene();
 
-				stats = new Stats();
-				container.appendChild( stats.dom );
+				updateColorEvent = { type: 'updateColor' };
 
-				// LIGHT
-				var ambientLight = new THREE.AmbientLight( 0x444444 );
-				scene.add( ambientLight );
+				var width = window.innerWidth;
+				var height = window.innerHeight;
+
+				perpCamera = new THREE.PerspectiveCamera( 60, width / height, 1, 100 );
+				perpCamera.position.set( 0, 0, 10 );
 
-				colorMap = 'rainbow';
-				numberOfColors = 512;
+				verticalLegendGroup = new THREE.Group();
+				horizontalLegendGroup = new THREE.Group();
 
-				legendLayout = 'vertical';
+				uiScene.add( verticalLegendGroup, horizontalLegendGroup );
+
+				// sized camera to legend dimenions
+				orthoCamera = new THREE.OrthographicCamera( - 6, 6, 2, - 2, 1, 10 );
+				orthoCamera.position.set( 3, 0, 10 );
+
+				var ambientLight = new THREE.AmbientLight( 0x444444 );
```

Comment:
One final suggestion: remove directional light, and add a point light to the camera. It will result in uniform lighting from all viewing directions.

Whether you include an ambient light, then, is up to you. If you include it, use this pattern,
```js
ambientLight = new THREE.AmbientLight( 0xffffff, intensity );
```

Reply:
Please read my suggestions carefully so we don't waste time.
```js
ambientLight = new THREE.AmbientLight( 0xffffff, 0.1 ); // or remove it
pointLight = new THREE.PointLight( 0xffffff, 0.9 );
scene.add( camera );
camera.add( pointLight );
```